sdb/commands/head.py

```diff
@@ -24,6 +24,15 @@
 
 
 class Head(sdb.Command):
+    """
+    Return the first COUNT objects passed in the pipeline
+
+    EXAMPLES
+        Print the name of the first pool
+
+            sdb> spa | head 1 | spa_name
```

Comment:
I think the command here would be `spa | head 1 | member spa_name` (the word `member` is missing above)